sycl/source/detail/device_impl.cpp

```diff
@@ -253,6 +253,21 @@ bool device_impl::has(aspect Aspect) const {
     return get_info<info::device::usm_restricted_shared_allocations>();
   case aspect::usm_system_allocator:
     return get_info<info::device::usm_system_allocator>();
+  case aspect::ext_intel_pci_address:
+  case aspect::ext_intel_gpu_eu_count:
+  case aspect::ext_intel_gpu_eu_simd_width:
+  case aspect::ext_intel_gpu_slices:
+  case aspect::ext_intel_gpu_subslices_per_slice:
+  case aspect::ext_intel_gpu_eu_count_per_subslice:
+    if (getPlugin().getBackend() == backend::level_zero) {
```

Comment:
Shouldn't we call get_info<...> here like it is done above? Some level zero or other implementations might not support some of these features.
OK if there is TODO saying that.

Reply:
@romanovvlad ,
These extensions are only supported on the Level Zero backend.  Any other backend should return false.
All of the new device descriptors access fields in structures that are part of the Level Zero Spec.  I do not anticipate them being removed.  This test should be sufficient.

Reply:
It would be nice if we could check if a particular backend supports these extensions. Can we make a call to the backend here and check if the result is PI_INVALID_VALUE? If it is, the code would return false, true otherwise.

Reply:
@romanovvlad , @smaslov-intel ,
As you know, in the level zero plugin, calls are wrapped to facilitate tracing.  As part of this facilitation, the error codes are checked.  If PI_INVALID_VALUE is returned, an exception is thrown from checkPiResult().   The control flow never returns to device_impl::has().  

The SYCL 2020 Provisional Spec has deprecated the info::device::extensions in favor of using info::device::aspects.  In the past, the info::device::extensions would have been used to determine if the platform supported the different capabilities.  This code is part of the aspect support.

Reply:
Can plugin::call_nocheck be used? This call should not call throw any exception in case of error code.

Reply:
The "nocheck" is actually a very logical counterpart for the "has" query.

Reply:
Done!  I did not know that call_nocheck() existed.  The support is more generic now.